python/katana/dynamic_bitset.pyx

```diff
@@ -0,0 +1,67 @@
+from .cpp.libgalois.DynamicBitset cimport DynamicBitset as CDynamicBitset
+
+
+cdef class DynamicBitset:
+    cdef CDynamicBitset underlying
+
+    def __cinit__(self):
+        self.underlying = CDynamicBitset()
```

Comment:
I don't think you need this. `underlying` is a value not a pointer so it will be default constructed anyway.